src/app/components/common/final-object.interface.ts

```diff
@@ -19,9 +19,14 @@ export interface ImageElement {
   fileName: string;              // for opening the file
   fileSize: number;              // file size in bytes
   hash: string;                  // used for detecting changed files and as a screenshot identifier
-  numOfScreenshots: number;      // number of screenshots for this file
+  height: number;                // height of the video
+  index: number;                 // for the `default` sort order
```

Comment:
Does this all get saved, and reserved into the vha file? 😕 

I'd prefer if we had a second object, or some other way where we don't save the things we can compute to the harddisk! What do you think? 💡

Reply:
After I submitted the PR and went to bed I realized the simplest thing to do is just have the `main.ts` (Node code) strip out all the fields we don't need to save 👍 

I'll make that change right now 🚀